client/components/DraftAddImage.jsx

```diff
@@ -2,6 +2,19 @@ import React, { useState } from "react";
 
 import { addNewBlock } from "medium-draft";
 
+const firebase = require("firebase/app");
+require("firebase/storage");
```

Comment:
import using ESM syntax ( `import ___ from ____`) rather than node syntax
also it sems like `firebase/storage` is unused?

Reply:
we need to import `firebase/storage` for `firebase.storage()` to work